publications/joss/paper.md

```diff
@@ -0,0 +1,129 @@
+---
+title: 'sbpy: A Python module for small-body planetary astronomy'
+tags:
+  - python
+  - astronomy
+  - solar system
+  - planetary science
+  - asteroids
+  - comets
+authors:
+  - name: Michael Mommert
+    orcid: 0000-0002-8132-778X
+    affiliation: 1
+  - name: Michael S. P. Kelley
+    orcid: 0000-0002-6702-7676
+    affiliation: 2
+  - name: Miguel de Val-Borro
+    orcid: 0000-0002-0455-9384	
+    affiliation: 3
+  - name: Jian-Yang Li
+    orcid: 0000-0003-3841-9977
+    affiliation: 3
+  - name: Giannina Guzman
+    orcid: 0000-0001-6340-8220
+    affiliation: 4
+  - name: Josef Durech
+    affiliation: 5
+  - name: Mikael Granvik
+    orcid: 0000-0002-5624-1888
+    affiliation: 6
+  - name: Will Grundy
+    orcid: 0000-0002-8296-6540
+    affiliation: 1
+  - name: Nick Moskovitz
+    orcid: 0000-0001-6765-6336
+    affiliation: 1
+  - name: Antti Penttilä
+    orcid: 0000-0001-7403-1721
+    affiliation: 6
+  - name: Nalin Samarasinha
+    orcid: 0000-0001-8925-7010
+    affiliation: 3
+affiliations:
+ - name: Lowell Observatory, US
+   index: 1
+ - name: University of Maryland, US
+   index: 2
+ - name: Planetary Science Institute, US
+   index: 3
+ - name: Villanova University, US
+   index: 4
+ - name: Charles Universiy, Prague, Czech Republic
+   index: 5
+ - name: University of Helsinki, Finland
+   index: 6
+
+date: 31 March 2019
+bibliography: paper.bib
+---
+
+# Summary
+
+Planetary astronomy - the study of Solar System objects with
+telescopic observations from the ground or from space - utilizes a
+wide range of methods that are common in observational
+astronomy. However, some aspects, including the planning of
+observations, as well as the analysis and interpretation of the
+results, require tailored techniques and models that are disparate
+from those used most other astronomical fields of research. Currently,
+there is no single open source software package available to support
+small-body planetary astronomers in their study of asteroids and
+comets in the same way in which ``Astropy`` [@astropy; @astropy2]
+supports the general astronomy community.
+
+``sbpy`` is a community effort to build a Python package for
+small-body planetary astronomy in the form of an ``Astropy` affiliated
```

Comment:
```
an ``Astropy` affiliated
```
should have double closing backtick characters

Reply:
Got it, thanks!